xpk/xpk.py

```diff
@@ -256,7 +256,10 @@ class SystemCharacteristics:
   chips_per_vm: int
 
 ################### Subcommand Helper Functions #############
-
+""" !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
@rwitten @gobbleturk, we plan to move xpk to its on repo (https://github.com/google/xpk) curious how we plan to support this connection in SystemCharacteristics? Can we make them independent ?

Reply:
Ideally MaxText and XPK would draw from the same source, but I'm unsure where such a source would live. I think we should discuss potential solutions

Reply:
For now, I think we can protect duplicated files with a clear comment. In the medium term we will add a max_utils meta package that all of these can depend on.